components/image-builder-mk3/pkg/orchestrator/orchestrator.go

```diff
@@ -345,12 +345,11 @@ func (o *Orchestrator) Build(req *protocol.BuildRequest, resp protocol.ImageBuil
 			Metadata: &wsmanapi.WorkspaceMetadata{
 				MetaId: buildID,
 				Annotations: map[string]string{
-					annotationRef:     wsrefstr,
-					annotationBaseRef: baseref,
+					annotationRef:       wsrefstr,
+					annotationBaseRef:   baseref,
+					annotationManagedBy: buildWorkspaceManagerID,
 				},
-				// TODO(cw): use the actual image build owner here and move to annotation based filter
-				//           when retrieving running image builds.
-				Owner: buildWorkspaceOwnerID,
+				Owner: req.GetTriggeredBy(),
```

Comment:
@Furisto Hm, I'm not sure I agree. Having the userId who triggered the image build might make sense to have for debugging purposes. But in every other way it's different to the owner of a regular or prebuild workspace.
What's the motivation to change this? :thinking:

Reply:
@geropl The motivation for this was that Prince and I were debugging imagebuilds last week and we found it difficult to identify which imagebuild we need to look at. I then implemented it like that because based on that comment there was already a plan to do it this way. I am open to do it differently e.g. keeping the image builder as the owner and adding a label or annotation with the user that triggered the imagebuild.

Reply:
IMHO it makes sense to make use the correct owner (i.e. the user originally triggering the image build). It's very much like prebuilds in that way.

Reply:
> So "who owns a prebuiöd/image build" is somewhat in a limbo state atm, at least that's how it feels to me.

Totally agree.